### PR TITLE
MPDX-6909: Coaching Answer Sets GraphQL

### PR DESCRIPTION
**Pull request by reldredge71**
Providing Coaching Answer Sets data using REST to GraphQL.

### CODE REVIEW

File: pages/api/Schema/CoachingAnswerSets/dataHandler.ts
Comment (OzzieOrca) [inline pages/api/Schema/CoachingAnswerSets/dataHandler.ts:67]: I think a `.map` would be better here.

File: pages/api/Schema/CoachingAnswerSets/dataHandler.ts
Comment (OzzieOrca) [inline pages/api/Schema/CoachingAnswerSets/dataHandler.ts:105]: I think you want the type condition in here also. There's a chance (very improbably) that you find an id that is from another type and stop looking for the one you really want too soon.

Same in your question function below.

**Review by OzzieOrca — CHANGES_REQUESTED** [review]
Looks pretty good :)

File: pages/api/Schema/CoachingAnswerSets/dataHandler.ts
Comment (TheNoodleMoose) [inline pages/api/Schema/CoachingAnswerSets/dataHandler.ts:75]: ```debugger``` here. Could probably squash the commit after removing it.

Comment (OzzieOrca) [reply pages/api/Schema/CoachingAnswerSets/dataHandler.ts:75]: 👍 PR looks good to me. I'm happy to approve it after this is removed :) Or anyone else can approve it.

**Review by TheNoodleMoose — APPROVED** [review]

**Pull request merged**
